### ISSUE TITLE
[Vinery] Server cannot start with vinery

### ISSUE THREAD

**Issue by Deadramas** [satisfyu/Let-s-Do-Hub#288]
### Minecraft Version

1.21.1

### Mod

Vinery

### Mod Version

1.5.0 Neoforge version

### Latest Version

- [X] Yes. I'm using the latest Version.

### Issue Description

Server cannot start. I updated my server with several mods with vinery. So the server was already running some time and i put vinery into it just now. Sadly it cannot start. not sure if its really a issue with your mod alone. But at least it does work without it. 

### Steps to Reproduce

Server start. 

### Logs

https://mclo.gs/QjsW4XL

### Minimal Instance Test

- [ ] Yes, I've tested on a minimal instance.

### Check for Similar Issues

- [X] Yes, I have checked for similar issues.

**Comment by Deadramas** [comment]
If you need some more informtion i am willing to help. Thank you very much:)

**Comment by TheoEN22** [comment]
Is it also stalling out after something about ServerSide Packet Registries? I think it might be the same issue that I posted about a few days ago [here](https://github.com/satisfyu/Let-s-Do-Hub/issues/281), if you want to take a look in case we can condense it down to just one GitHub thread.

**Comment by Deadramas** [comment]
Tbh i have no idea if thats the problem. Could be.

When i start a older world in a single player world. it works. So maybe you are right.

**Comment by Deadramas** [comment]
> Is it also stalling out after something about ServerSide Packet Registries? I think it might be the same issue that I posted about a few days ago [here](https://github.com/satisfyu/Let-s-Do-Hub/issues/281), if you want to take a look in case we can condense it down to just one GitHub thread.

I opened the sever world with all mods including vinery in a singleplayer world and it worked there. just not on the server, so maybe you are right.

**Comment by freddthedead** [comment]
I am having the same issue. I initially thought it was the API and posted this over there, but the server starts OK with the API mod, just not Vinery mod.  It works fine in singleplayer, just not on the server.

I am using Fabric Loader 0.16.7 for MC1.21.1. Mods installed are:
architecture-13.0.8 fabric
fabric-api-0.106
lets-API-fabric-1.3.0 beta
letsdo-vinery-fabric-1.5.0 beta

Here is the crash log:  

26.10 17:04:22 [Multicraft] Starting server!
26.10 17:04:23 [Multicraft] Updating eula.txt file
26.10 17:04:23 [Server] INFO WARNING: Published ports are discarded when using host network mode
26.10 17:04:23 [Server] INFO ec557fde4183f2cec75212e3751978605051e8b61a2b24eb80f9001e0f360264
26.10 17:04:24 [Server] INFO Starting net.fabricmc.loader.impl.game.minecraft.BundlerClassPathCapture
26.10 17:04:24 [Server] INFO [Log4jPatcher] [INFO] Transforming org/apache/logging/log4j/core/pattern/MessagePatternConverter
26.10 17:04:24 [Server] INFO [Log4jPatcher] [WARN]  Unable to find noLookups:Z field in org/apache/logging/log4j/core/pattern/MessagePatternConverter
26.10 17:04:24 [Server] INFO [Log4jPatcher] [INFO] Transforming org/apache/logging/log4j/core/lookup/JndiLookup
26.10 17:04:25 [Server] main/INFO Loading Minecraft 1.21.1 with Fabric Loader 0.16.7
26.10 17:04:25 [Server] main/INFO Loading 46 mods:
26.10 17:04:25 [Server] INFO - architectury 13.0.8
26.10 17:04:25 [Server] INFO - doapi 1.3.0-beta-release
26.10 17:04:25 [Server] INFO |-- blue_endless_jankson 1.2.3
26.10 17:04:25 [Server] INFO \-- terraform-wood-api-v1 11.0.0-alpha.1
26.10 17:04:25 [Server] INFO - fabric-api 0.106.0+1.21.1
26.10 17:04:25 [Server] INFO |-- fabric-api-base 0.4.42+6573ed8c19
26.10 17:04:25 [Server] INFO |-- fabric-api-lookup-api-v1 1.6.68+b559734419
26.10 17:04:25 [Server] INFO |-- fabric-biome-api-v1 13.0.29+5bd9f1bc19
26.10 17:04:25 [Server] INFO |-- fabric-block-api-v1 1.0.22+0af3f5a719
26.10 17:04:25 [Server] INFO |-- fabric-block-view-api-v2 1.0.10+6573ed8c19
26.10 17:04:25 [Server] INFO |-- fabric-blockrenderlayer-v1 1.1.52+0af3f5a719
26.10 17:04:25 [Server] INFO |-- fabric-client-tags-api-v1 1.1.15+6573ed8c19
26.10 17:04:25 [Server] INFO |-- fabric-command-api-v1 1.2.49+f71b366f19
26.10 17:04:25 [Server] INFO |-- fabric-command-api-v2 2.2.28+6ced4dd919
26.10 17:04:25 [Server] INFO |-- fabric-commands-v0 0.2.66+df3654b319
26.10 17:04:25 [Server] INFO |-- fabric-content-registries-v0 8.0.16+b559734419
26.10 17:04:25 [Server] INFO |-- fabric-convention-tags-v1 2.1.0+7f945d5b19
26.10 17:04:25 [Server] INFO |-- fabric-convention-tags-v2 2.9.0+83a9d31319
26.10 17:04:25 [Server] INFO |-- fabric-crash-report-info-v1 0.2.29+0af3f5a719
26.10 17:04:25 [Server] INFO |-- fabric-data-attachment-api-v1 1.1.28+1daea21519
26.10 17:04:25 [Server] INFO |-- fabric-data-generation-api-v1 20.2.21+16c4ae2519
26.10 17:04:25 [Server] INFO |-- fabric-dimensions-v1 4.0.0+6fc22b9919
26.10 17:04:25 [Server] INFO |-- fabric-entity-events-v1 1.7.0+2122d82819
26.10 17:04:25 [Server] INFO |-- fabric-events-interaction-v0 0.7.13+ba9dae0619
26.10 17:04:25 [Server] INFO |-- fabric-game-rule-api-v1 1.0.53+6ced4dd919
26.10 17:04:25 [Server] INFO |-- fabric-item-api-v1 11.1.0+d5debaed19
26.10 17:04:25 [Server] INFO |-- fabric-item-group-api-v1 4.1.5+3fc0e55219
26.10 17:04:25 [Server] INFO |-- fabric-key-binding-api-v1 1.0.47+0af3f5a719
26.10 17:04:25 [Server] INFO |-- fabric-keybindings-v0 0.2.45+df3654b319
26.10 17:04:25 [Server] INFO |-- fabric-lifecycle-events-v1 2.3.12+6c1df36019
26.10 17:04:25 [Server] INFO |-- fabric-loot-api-v2 3.0.14+3f89f5a519
26.10 17:04:25 [Server] INFO |-- fabric-loot-api-v3 1.0.2+3f89f5a519
26.10 17:04:25 [Server] INFO |-- fabric-message-api-v1 6.0.13+6573ed8c19
26.10 17:04:25 [Server] INFO |-- fabric-model-loading-api-v1 2.0.0+fe474d6b19
26.10 17:04:25 [Server] INFO |-- fabric-networking-api-v1 4.3.0+c7469b2119
26.10 17:04:25 [Server] INFO |-- fabric-object-builder-api-v1 15.2.0+40875a9319
26.10 17:04:25 [Server] INFO |-- fabric-particles-v1 4.0.2+6573ed8c19
26.10 17:04:25 [Server] INFO |-- fabric-recipe-api-v1 5.0.13+6508971219
26.10 17:04:25 [Server] INFO |-- fabric-registry-sync-v0 5.1.3+60c3209b19
26.10 17:04:25 [Server] INFO |-- fabric-renderer-api-v1 3.4.0+c705a49c19
26.10 17:04:25 [Server] INFO |-- fabric-renderer-indigo 1.7.0+c705a49c19
26.10 17:04:25 [Server] INFO |-- fabric-renderer-registries-v1 3.2.68+df3654b319
26.10 17:04:25 [Server] INFO |-- fabric-rendering-data-attachment-v1 0.3.48+73761d2e19
26.10 17:04:25 [Server] INFO |-- fabric-rendering-fluids-v1 3.1.6+1daea21519
26.10 17:04:25 [Server] INFO |-- fabric-rendering-v0 1.1.71+df3654b319
26.10 17:04:25 [Server] INFO |-- fabric-rendering-v1 5.0.5+df16efd019
26.10 17:04:25 [Server] INFO |-- fabric-resource-conditions-api-v1 4.3.0+8dc279b119
26.10 17:04:25 [Server] INFO |-- fabric-resource-loader-v0 1.3.0+5659912919
26.10 17:04:25 [Server] INFO |-- fabric-screen-api-v1 2.0.25+8b68f1c719
26.10 17:04:25 [Server] INFO |-- fabric-screen-handler-api-v1 1.3.87+b559734419
26.10 17:04:25 [Server] INFO |-- fabric-sound-api-v1 1.0.23+6573ed8c19
26.10 17:04:25 [Server] INFO |-- fabric-transfer-api-v1 5.4.0+95bead5f19
26.10 17:04:25 [Server] INFO \-- fabric-transitive-access-wideners-v1 6.1.0+1daea21519
26.10 17:04:25 [Server] INFO - fabricloader 0.16.7
26.10 17:04:25 [Server] INFO \-- mixinextras 0.4.1
26.10 17:04:25 [Server] INFO - java 21
26.10 17:04:25 [Server] INFO - minecraft 1.21.1
26.10 17:04:25 [Server] INFO - vinery 1.5.0-beta-release
26.10 17:04:25 [Server] main/INFO SpongePowered MIXIN Subsystem Version=0.8.7 Source=file:/jar/libraries/net/fabricmc/sponge-mixin/0.15.3+mixin.0.8.7/sponge-mixin-0.15.3+mixin.0.8.7.jar Service=Knot/Fabric Env=SERVER
26.10 17:04:25 [Server] main/INFO Compatibility level set to JAVA_16
26.10 17:04:25 [Server] main/INFO Compatibility level set to JAVA_17
26.10 17:04:26 [Server] main/INFO Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.4.1).
26.10 17:04:32 [Server] main/ERROR No data fixer registered for doapi:chair
26.10 17:04:32 [Server] main/ERROR Failed to start the minecraft server
26.10 17:04:32 [Server] INFO java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'doapi' at 'de.cristelknight.doapi.fabric.DoApiFabric'!
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:403) ~[fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:401) ~[fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:63) ~[fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.minecraft.server.Main.main(Main.java:111) [server-intermediary.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) [fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) [fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO Suppressed: java.lang.NoClassDefFoundError: Could not initialize class net.satisfy.vinery.registry.ObjectRegistry
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.Vinery.init(Vinery.java:29) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.fabric.VineryFabric.onInitialize(VineryFabric.java:12) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:399) ~[fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:63) ~[fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.minecraft.server.Main.main(Main.java:111) [server-intermediary.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) [fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) [fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NoClassDefFoundError: net/minecraft/class_638 [in thread "main"]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.ObjectRegistry.lambda$static$80(ObjectRegistry.java:145) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/dev.architectury.registry.registries.fabric.RegistrarManagerImpl$RegistrarImpl.register(RegistrarManagerImpl.java:232) ~[architectury-13.0.8-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.Util.registerItem(Util.java:50) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.ObjectRegistry.registerItem(ObjectRegistry.java:345) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.ObjectRegistry.<clinit>(ObjectRegistry.java:145) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.StorageTypeRegistry.registerBlocks(StorageTypeRegistry.java:22) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.util.api.VineryDoApi.registerBlocks(VineryDoApi.java:22) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.common.registry.DoApiBlockEntityTypes.getBlocksForEntity(DoApiBlockEntityTypes.java:45) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.common.registry.DoApiBlockEntityTypes.lambda$static$0(DoApiBlockEntityTypes.java:24) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/dev.architectury.registry.registries.fabric.RegistrarManagerImpl$RegistrarImpl.register(RegistrarManagerImpl.java:232) ~[architectury-13.0.8-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/dev.architectury.registry.registries.DeferredRegister.register(DeferredRegister.java:78) ~[architectury-13.0.8-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.common.registry.DoApiBlockEntityTypes.init(DoApiBlockEntityTypes.java:55) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.DoApi.init(DoApi.java:15) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.fabric.DoApiFabric.onInitialize(DoApiFabric.java:10) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO ... 7 more
26.10 17:04:32 [Server] INFO Caused by: java.lang.NoClassDefFoundError: net/minecraft/class_638
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.ObjectRegistry.lambda$static$80(ObjectRegistry.java:145) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/dev.architectury.registry.registries.fabric.RegistrarManagerImpl$RegistrarImpl.register(RegistrarManagerImpl.java:232) ~[architectury-13.0.8-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.Util.registerItem(Util.java:50) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.ObjectRegistry.registerItem(ObjectRegistry.java:345) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.ObjectRegistry.<clinit>(ObjectRegistry.java:145) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.StorageTypeRegistry.registerBlocks(StorageTypeRegistry.java:22) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.util.api.VineryDoApi.registerBlocks(VineryDoApi.java:22) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.common.registry.DoApiBlockEntityTypes.getBlocksForEntity(DoApiBlockEntityTypes.java:45) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.common.registry.DoApiBlockEntityTypes.lambda$static$0(DoApiBlockEntityTypes.java:24) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/dev.architectury.registry.registries.fabric.RegistrarManagerImpl$RegistrarImpl.register(RegistrarManagerImpl.java:232) ~[architectury-13.0.8-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/dev.architectury.registry.registries.DeferredRegister.register(DeferredRegister.java:78) ~[architectury-13.0.8-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.common.registry.DoApiBlockEntityTypes.init(DoApiBlockEntityTypes.java:55) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.DoApi.init(DoApi.java:15) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.fabric.DoApiFabric.onInitialize(DoApiFabric.java:10) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:399) ~[fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO ... 6 more
26.10 17:04:32 [Server] INFO Caused by: java.lang.ClassNotFoundException: net.minecraft.class_638
26.10 17:04:32 [Server] INFO at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
26.10 17:04:32 [Server] INFO at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226) ~[fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.ObjectRegistry.lambda$static$80(ObjectRegistry.java:145) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/dev.architectury.registry.registries.fabric.RegistrarManagerImpl$RegistrarImpl.register(RegistrarManagerImpl.java:232) ~[architectury-13.0.8-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.Util.registerItem(Util.java:50) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.ObjectRegistry.registerItem(ObjectRegistry.java:345) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.ObjectRegistry.<clinit>(ObjectRegistry.java:145) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.registry.StorageTypeRegistry.registerBlocks(StorageTypeRegistry.java:22) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/net.satisfy.vinery.util.api.VineryDoApi.registerBlocks(VineryDoApi.java:22) ~[letsdo-vinery-fabric-1.5.0-beta-release.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.common.registry.DoApiBlockEntityTypes.getBlocksForEntity(DoApiBlockEntityTypes.java:45) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.common.registry.DoApiBlockEntityTypes.lambda$static$0(DoApiBlockEntityTypes.java:24) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/dev.architectury.registry.registries.fabric.RegistrarManagerImpl$RegistrarImpl.register(RegistrarManagerImpl.java:232) ~[architectury-13.0.8-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/dev.architectury.registry.registries.DeferredRegister.register(DeferredRegister.java:78) ~[architectury-13.0.8-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.common.registry.DoApiBlockEntityTypes.init(DoApiBlockEntityTypes.java:55) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.DoApi.init(DoApi.java:15) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at knot/de.cristelknight.doapi.fabric.DoApiFabric.onInitialize(DoApiFabric.java:10) ~[letsdo-API-fabric-1.3.0-beta-release-fabric.jar:?]
26.10 17:04:32 [Server] INFO at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:399) ~[fabric-loader-0.16.7.jar:?]
26.10 17:04:32 [Server] INFO ... 6 more
26.10 17:04:33 [Multicraft] Server shut down (starting)
26.10 17:04:33 [Multicraft] Restarting crashed server in 300 seconds
26.10 17:04:33 [Multicraft] Server stopped

**Comment by TheDanilux** [comment]
Hello, I am having the same issue too. Like others, when i remove vinery, the server start as exepted. I'm using the latest fabric for 1.21.1 and the latest beta version for 1.21.1 of doapi and letsdo vinery.
Here's the crash log:
https://mclo.gs/JgQtLcP

**Comment by Zelioth** [comment]
Opened an issue but it was closed as completed. I didn't see this one at first so I apologize. I ran up the main three mods in a controlled instance and it came with this log.  All of the latest updates and neoforge updated. 

[latest.log](https://github.com/user-attachments/files/17596234/latest.log)
[debug.log](https://github.com/user-attachments/files/17596236/debug.log)

**Comment by FelixAgairu** [comment]
FaC works, but Vinery not.
[server.log](https://github.com/user-attachments/files/17609956/2024-11-03-7.log)

**Comment by CFood0112** [comment]
I am experiencing the same issue. To clarify, I'm using the Fabric version, with both the server and client running version 1.21. After installing Vinery on the server, my server fails to start. Initially, I suspected that this was due to a mismatch between the game version and the mod version. However, since 1.21.1 does not introduce any substantial updates over 1.21, and the mod runs perfectly on a 1.21 client, I ruled out this possibility.

I tried uninstalling the mod while keeping the required API, and the server started without any issues. At first, I thought the issue might be due to conflicts with other mods, but since many users are facing the same problem, I dismissed that idea. I really enjoy Vinery and sincerely hope this issue can be resolved soon so everyone can continue to enjoy the mod.
[latest.log](https://github.com/user-attachments/files/17667064/latest.log)
